apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/navigation-menu/index.php

```diff
@@ -85,11 +86,14 @@ function render_navigation_menu_block( $attributes ) {
 	ob_start();
 	// phpcs:disable WordPress.WP.I18n.NonSingularStringLiteralText
 	?>
-	<nav class="<?php echo esc_attr( $wrapper_attr['class'] ); ?>" aria-label="<?php esc_attr_e( $wrapper_attr['label'], 'twentynineteen' ); ?>">
-		<?php
-		wp_nav_menu( get_menu_params_by_theme_location( $location ) );
-		?>
-	</nav><!-- #site-navigation -->
+	<div class="wp-block-a8c-navigation-menu">
```

Comment:
This adds extra markup to the front-end, which we should avoid; could we conditionally include it in the `nav` class below? Otherwise, if there's no way to reconcile markup/classes for both the front-end and the edit view, we could have two different rendering functions, like the [site logo](https://github.com/Automattic/wp-calypso/blob/master/apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-logo/index.php).

Reply:
Was having trouble yesterday with a difference in the nesting of classes in the main theme and our blocks - will take another look at it and sort out a better way of reconciling the two.

Reply:
https://github.com/Automattic/wp-calypso/pull/34396/commits/2a4a0114135fe27a0c13eec43a6325915b366910 removes the extra wrapper and instead modifies the css selectors to make the correct styles apply with both classes on the same element - is this a better approach?

Reply:
Awesome, looks good!